cli/command/plugin/install.go

```diff
@@ -18,16 +18,20 @@ type pluginOptions struct {
 	name       string
 	grantPerms bool
 	disable    bool
+	args       []string
 }
 
 func newInstallCommand(dockerCli *command.DockerCli) *cobra.Command {
 	var options pluginOptions
 	cmd := &cobra.Command{
-		Use:   "install [OPTIONS] PLUGIN",
+		Use:   "install [OPTIONS] PLUGIN [key=value...]",
```

Comment:
I even think it should be `[NAME[.FIELD][=VALUE]...]` and same in set. WDYT @tiborvass ?